{"name": "test_login_with_invalid_credentials", "status": "passed", "steps": [{"name": "Вход в систему с недействительными учетными данными", "status": "passed", "steps": [{"name": "Регистрация ответа", "status": "passed", "attachments": [{"name": "Текст ответа", "source": "575f0071-3021-455e-aecb-8fd48eba15f2-attachment.json", "type": "application/json"}, {"name": "Заголовки", "source": "529e79ce-01b1-4870-b0a0-2b2431d354f0-attachment.json", "type": "application/json"}], "start": 1719732381516, "stop": 1719732381517}], "start": 1719732380403, "stop": 1719732381517}], "start": 1719732380403, "stop": 1719732381518, "uuid": "efcddd19-03aa-4b5f-a34c-d68331c3c222", "historyId": "15d5d744ddd2cc4bbf96e98f1487c81f", "testCaseId": "15d5d744ddd2cc4bbf96e98f1487c81f", "fullName": "tests.test_user_operations.TestUserOperations#test_login_with_invalid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('user_page', 'user_data', 'created_user')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_operations"}, {"name": "subSuite", "value": "TestUserOperations"}, {"name": "host", "value": "sanya"}, {"name": "thread", "value": "8824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_operations"}]}